src/Validation/Validation.php

```diff
@@ -1308,12 +1337,16 @@ public static function imageHeight(array $file, string $operator, int $height):
      * - `format` - By default `both`, can be `long` and `lat` as well to validate
      *   only a part of the coordinate.
      *
-     * @param string $value Geographic location as string
+     * @param mixed $value Geographic location as string
      * @param array $options Options for the validation logic.
      * @return bool
      */
-    public static function geoCoordinate(string $value, array $options = []): bool
+    public static function geoCoordinate($value, array $options = []): bool
     {
+        if (!is_scalar($value)) {
```

Comment:
Do we have existing tests for these scenarios now that they don't create `TypeError` again?